code/modules/projectiles/guns/projectile/boltgun/anti_materiel_sniper.dm

```diff
@@ -17,7 +17,7 @@
 	fire_sound = 'sound/weapons/guns/fire/sniper_fire.ogg'
 	reload_sound 	= 'sound/weapons/guns/interact/rifle_load.ogg'
 	matter = list(MATERIAL_PLASTEEL = 40, MATERIAL_PLASTIC = 20, MATERIAL_DIAMOND = 3, MATERIAL_OSMIUM = 5, MATERIAL_URANIUM = 2)
-	price_tag = 10000
+	price_tag = 5000
```

Comment:
```suggestion
	price_tag = 7500
```
Its insainly rare of a get takes a lot to make make and the few natural cases to get it (uplinks) are limited a lot.
5k is good but not enuff to sell imo